### DISCUSSION TITLE
Is it possible to run two distinct seed scripts in Prisma separately?

### DISCUSSION THREAD

**Discussion by rupam-kundu-itobuz** [prisma/prisma#10594] [Q&A] [answered]
I want to run two different seed scripts for populating my database with mock data separately. Is this possible? How can I accomplish this in case it is possible?

**Comment by janpio** [comment]
What exactly do you mean by `run two different seed scripts ... separately`? What is your use case?

**Reply by rupam-kundu-itobuz** [reply]
Hi @janpio. Thanks for replying.

I want one seed script to populate data essential to running the application and the other to populate non essential mock data. I need to be able to run either seed script by running a command. It is okay if the two commands are different. The two seed scripts should be able to run separately on demand by running a command in each case. Is this possible in Prisma?

**Reply by janpio** [reply, marked as answer]
That is possible completely outside the scope of Prisma. The 1 script you define to run via `prisma seed` is only really required for where Prisma automatically runs seeding (in your case probably the essential data) on other commands. As the command itself always is also be able to be run outside of Prisma, with e.g. `ts-node seed.ts` or similar, you can just have a second script for the mock data.

**Comment by sharun-oren** [comment]
I have created a script for running multiple seed files 
**prisma/seed.ts**
```
const { PrismaClient } = require("@prisma/client");
const fs = require("fs");
const path = require("path");

const prisma = new PrismaClient();

async function seed() {
  // add path for seed files
  const seedFilesPath = path.join(__dirname);

  const seedFiles = fs
    .readdirSync(seedFilesPath)
    .filter((file: string) => file.endsWith(".seed.ts"));

  for (const seedFile of seedFiles) {
    const seedFilePath = path.join(seedFilesPath, seedFile);
    const { default: seedFunction } = require(seedFilePath);
    await seedFunction(prisma);
  }

  console.log("Seeding completed successfully");
}

seed()
  .catch((error) => {
    console.error("Seeding error:", error);
  })
  .finally(async () => {
    await prisma.$disconnect();
  });

```
Just need to add files like `[file_name].seed.ts` in which file_name is your file name 
in package.json add seed file as the given file

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
